rs/nns/handlers/root/impl/canister/canister.rs

```diff
@@ -282,13 +232,9 @@ async fn update_canister_settings_(
 }
 
 /// Resources to serve for a given http_request
-#[export_name = "canister_query http_request"]
-fn http_request() {
-    over(candid_one_with_config, serve_http)
-}
-
 /// Serve an HttpRequest made to this canister
-pub fn serve_http(request: HttpRequest) -> HttpResponse {
+#[query(hidden = true, decoding_quota = 10000)]
```

Comment:
It seems that we were not been enforcing the decoding quota before, and are going to now. That seems fine (especially given that this is an http request, so I doubt this would break any clients), just wanted to make sure this is a conscious decision.

Reply:
Closing as non-blocking.

Reply:
We had a quota before, but it was enforced differently. `over(candid_one_with_config, serve_http)` checks the quota on the objects defined as an interface.